docs/streaming-custom-receivers.md

```diff
@@ -257,9 +257,9 @@ The following table summarizes the characteristics of both types of receivers
 
 ## Implementing and Using a Custom Actor-based Receiver
 
-Custom [Akka Actors](http://doc.akka.io/docs/akka/2.2.4/scala/actors.html) can also be used to
+Custom [Akka Actors](http://doc.akka.io/docs/akka/current/scala/actors.html) can also be used to
```

Comment:
linking to current seem like a bad idea given they could have api breaking changes?